coreneuron/utils/randoms/nrnran123.cu

```diff
@@ -95,9 +95,7 @@ CORENRN_DEVICE philox4x32_key_t* g_k_dev;
 
 OMP_Mutex g_instance_count_mutex;
 
-nrn_pragma_omp(declare target)
```

Comment:
makes sense - I think that was left from an earlier attempt where the functions using this variable were also annoted.